### DISCUSSION TITLE
Simultaneous Charging Sessions

### DISCUSSION THREAD

**Discussion by arahman33** [citrineos/citrineos#43] [General]
Hi everyone,

Is there anyone that has been able to start multiple transactions, i.e., starting a transaction with multiple charge points? Currently, I am getting the concurrent TX error. If anyone has been able to achieve this, please advise on setting the environment that will enable multiple transactions.

Thanks!

**Comment by sethunath** [comment]
please check the idTag that you are using.if you are using the same idTag for both transactions, the idTag should allow concurrent transactions
